introduction.md

```diff
@@ -1,2 +1,8 @@
 Hello Dojo
 favorite Code School path - JavaScript
+Skills:
+* JavaScript
+* Git
+* PHP
+* HTML
+* CSS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?